Add information on recovery and safe mode (#31427)
home-assistant/home-assistant.github.io: source/_integrations/recovery_mode.markdown

ha_integration_type: system
---

The **Recovery mode** integration is an internal integration used by the
Home Assistant Core.

> You don't have to configure it in any way since it is automatically always

Reviewer: ```suggestion
You don't have to configure it since it is automatically always
```

> it will still continue to start using bits and pieces from the configuration

Reviewer: ```suggestion
it will still continue to start using parts of the configuration
```

> of the last time it did start.

Reviewer: ```suggestion
from the last time Home Assistant did start.
```

When this happens, Home Assistant will start in **Recovery mode** using this

> integration. In this mode, nothing is loaded, but it does give you access to

Reviewer: ```suggestion
integration. In this mode, no user configured integrations are loaded, but it does give you access to
```
